app/mailers/twilio_sender.rb

```diff
@@ -2,23 +2,96 @@
+      monitoree_number = Phonelib.parse(phone_numbers[:monitoree_number], 'US').full_e164
+      BlockedNumber.create(phone_number: monitoree_number) unless BlockedNumber.exists?(phone_number: monitoree_number)
+    # Invalid Mobile Number Error https://www.twilio.com/docs/api/errors/21614
+    when @twilio_error_codes[:invalid_number]
+      dispatch_errored_contact_history_items(patient, 'Invalid recipient phone number.')
+    # Unsupported Region Error https://www.twilio.com/docs/api/errors/21408
+    when @twilio_error_codes[:unsupported_region]
+      dispatch_errored_contact_history_items(patient, 'Recipient phone number is in an unsupported region.')
+    # Unreachable Destination Handset Error https://www.twilio.com/docs/api/errors/30003
+    when @twilio_error_codes[:unreachable_unavailable]
+      error_message = 'Recipient phone is off, may not be eligible to receive SMS messages, or is otherwise unavailable.'
+      dispatch_errored_contact_history_items(patient, error_message)
+    # Message Blocked Error https://www.twilio.com/docs/api/errors/30004
+    when @twilio_error_codes[:unavailable_ineligible]
+      error_message = 'Recipient may have blocked communications with SaraAlert, recipient phone may be unavailable or ineligible to receive SMS text messages.'
+      dispatch_errored_contact_history_items(patient, error_message)
+    # Unknown Destination Handset Error https://www.twilio.com/docs/api/errors/30005
+    when @twilio_error_codes[:non_existent_or_off]
+      error_message = 'Recipient phone number may not exist, the phone may be off or the phone is not eligible to receive SMS text messages.'
+      dispatch_errored_contact_history_items(patient, error_message)
+    # Landline or Unreachable Carrier Error https://www.twilio.com/docs/api/errors/30006
+    when @twilio_error_codes[:sms_ineligible]
+      error_message = 'Recipient phone number may not eligible to receive SMS text messages, or carrier network may be unreachable.'
+      dispatch_errored_contact_history_items(patient, error_message)
+    # Message Filtered By Carrier Error https://www.twilio.com/docs/api/errors/30007
+    when @twilio_error_codes[:carrier_filter]
+      dispatch_errored_contact_history_items(patient, 'Message has been filtered by carrier network.')
+    # Unknown Error https://www.twilio.com/docs/api/errors/30008
+    when @twilio_error_codes[:unknown_error]
+      error_message = 'An unknown error has been encountered by the messaging system. '
+      error_message += 'The system will retry in an hour if it is still in monitoree’s preferred contact period.'
+      dispatch_errored_contact_history_items(patient, error_message)
+    else
+      dispatch_errored_contact_history_items(patient, 'An unknown error has been encountered by the messaging system.')
+    end
+  end
+
+  class << self
+    attr_reader :twilio_error_codes
```

Comment:
Why just just make the error codes a class constant? Also not clear on why this class is necessary